### PR TITLE
add support for classifiers and mappers

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24308
fixes: https://github.com/demisto/etc/issues/24564

## Description
Added support for new classifiers and mappers

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -298,7 +298,14 @@
 PACKS_PLAYBOOK_YML_REGEX = r'{}{}/([^/]+)/{}/([^.]+)\.yml'.format(CAN_START_WITH_DOT_SLASH, PACKS_DIR, PLAYBOOKS_DIR)
 PACKS_TEST_PLAYBOOKS_REGEX = r'{}{}/([^/]+)/{}/([^.]+)\.yml'.format(CAN_START_WITH_DOT_SLASH, PACKS_DIR,
                                                                     TEST_PLAYBOOKS_DIR)
-PACKS_CLASSIFIERS_REGEX = r'{}{}/([^/]+)/{}/([^.]+)\.json'.format(CAN_START_WITH_DOT_SLASH, PACKS_DIR, CLASSIFIERS_DIR)
+PACKS_CLASSIFIERS_REGEX = r'{}{}/([^/]+)/{}/*classifier-(?!mapper).*(?<!5_9_9)\.json'.format(
```
Comment (ronykoz) [inline demisto_sdk/commands/common/constants.py:301]: why do we need the 599 there?

File: demisto_sdk/commands/common/hook_validations/classifier.py [hunk 2]
```diff
@@ -1,12 +1,119 @@
+from distutils.version import LooseVersion
+
+from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
     ContentEntityValidator
 
+FROM_VERSION_FOR_NEW_CLASSIFIER = '6.0.0'
+TO_VERSION_FOR_OLD_CLASSIFIER = '5.9.9'
+VALID_TYPE = 'classification'
```
Comment (ronykoz) [inline demisto_sdk/commands/common/hook_validations/classifier.py:9]:
```suggestion
CLASSIFICATION_TYPE = 'classification'
```

**Comment by ronykoz** [comment]
@Itay4 please go over the schemes

File: demisto_sdk/commands/upload/uploader.py [hunk 3]
```diff
@@ -62,6 +62,8 @@ def upload(self):
                 self.incident_type_uploader(self.path)
             elif file_type == 'classifier':
                 self.classifier_uploader(self.path)
+            elif file_type == 'classifier_5_9_9':
```
[the remark this answers is not in the document]
Comment (barchen1) [reply demisto_sdk/commands/upload/uploader.py:65]: will open an issue about it.

File: demisto_sdk/commands/common/schemas/mapper.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+type: map
+mapping:
```
Comment (Itay4) [inline demisto_sdk/commands/common/schemas/mapper.yml:2]: lets add `feed` field here as well

**Comment by Itay4** [comment]
also need to modify integration schema to support default classifier/mapper fields

File: demisto_sdk/commands/common/schemas/mapper.yml (hunk 4 above)
Comment (barchen1) [reply demisto_sdk/commands/common/schemas/mapper.yml:2]: done

**Comment by barchen1** [comment]
@ronykoz I run sdk with the new changes on master and it failed. 
We need to merge this pr first https://github.com/demisto/content/pull/7065 and then it will pass.
@Itay4 FYI

**Comment by Itay4** [comment]
schemas look good

**Comment by ronykoz** [comment]
This PR will break content until content restructure, force merging

**Pull request merged**